fairlearn/reductions/grid_search/grid_search.py

```diff
@@ -187,3 +210,41 @@ def _regression_weight_function(self, a_val, trade_off, p0, p1, a0_val):
             return trade_off / p0
         else:
             return (1 - trade_off) / p1
+
+    def _make_vector(self, formless, formless_name):
+        formed_vector = None
+        if isinstance(formless, pd.DataFrame):
+            if len(formless.columns) == 1:
+                formed_vector = formless[0].to_numpy()
+            else:
+                raise RuntimeError("%s is a DataFrame with more than one column" % formless_name)
```

Comment:
Also, as a nit, our other codebases (as well as AzureMlCli) have largely standardized around using `"".format()`

Reply:
I tend to use that as well, since you don't need to actually care about making the argument a string. format() takes care of that.

Reply:
Changed to match